packages/features/eventtypes/components/tabs/advanced/EventAdvancedTab.tsx

```diff
@@ -247,125 +504,25 @@ export const EventAdvancedTab = ({
-                  className="hover:stroke-3 hover:text-emphasis min-w-fit !py-0 px-0 hover:bg-transparent"
-                  onClick={() => setShowEventNameTip((old) => !old)}>
-                  <Icon name="pencil" className="h-4 w-4" />
-                </Button>
-              }
-            />
-          </div>
-        </div>
-        <div className="space-y-2">
-          {displayDestinationCalendarSelector && (
-            <div className={classNames("w-full", customClassNames?.addToCalendarEmailOrganizer?.container)}>
-              <Switch
-                tooltip={t("if_enabled_email_address_as_organizer")}
-                label={
-                  <>
-                    {t("display_add_to_calendar_organizer")}
-                    <Icon
-                      name="info"
-                      className="text-default hover:text-attention hover:bg-attention ms-1 inline h-4 w-4 rounded-md"
-                    />
-                  </>
-                }
-                checked={useEventTypeDestinationCalendarEmail}
-                onCheckedChange={(val) => {
-                  setUseEventTypeDestinationCalendarEmail(val);
-                  formMethods.setValue("useEventTypeDestinationCalendarEmail", val, { shouldDirty: true });
-                  if (val) {
-                    showToast(t("reconnect_calendar_to_use"), "warning");
-                  }
-                }}
-              />
-            </div>
-          )}
-          {!useEventTypeDestinationCalendarEmail && verifiedSecondaryEmails.length > 0 && !team && (
-            <div className={cx("flex w-full flex-col", displayDestinationCalendarSelector && "pl-11")}>
-              <SelectField
-                placeholder={
-                  selectedSecondaryEmailId === -1 && (
-                    <span className="text-default min-w-0 overflow-hidden truncate whitespace-nowrap">
-                      <Badge variant="blue">{t("default")}</Badge> {userEmail}
-                    </span>
-                  )
-                }
-                className={customClassNames?.addToCalendarEmailOrganizer?.emailSelect?.select}
-                containerClassName={customClassNames?.addToCalendarEmailOrganizer?.emailSelect?.container}
-                onChange={(option) =>
-                  formMethods.setValue("secondaryEmailId", option?.value, { shouldDirty: true })
-                }
-                value={verifiedSecondaryEmails.find(
-                  (secondaryEmail) =>
-                    selectedSecondaryEmailId !== -1 && secondaryEmail.value === selectedSecondaryEmailId
-                )}
-                options={verifiedSecondaryEmails}
-              />
-              <p
-                className={classNames(
-                  "text-subtle mt-2 text-sm",
-                  customClassNames?.addToCalendarEmailOrganizer?.emailSelect?.displayEmailLabel
-                )}>
-                {t("display_email_as_organizer")}
-              </p>
-            </div>
```

Comment:
Moved all of it to `CalendarSettings` and also supported showing a Skeleton till these settings load instead of not showing anything and some UI suddenly appearing in between.